[Guide updates] Simplify labels guide (#34742)
fleetdm/fleet: articles/managing-labels-in-fleet.md

![Managing labels in Fleet](../website/assets/images/articles/managing-labels-in-fleet-1600x900@2x.png)

> In Fleet you can labels (tags), to scope [software](https://fleetdm.com/guides/deploy-software-packages), [policies](https://fleetdm.com/securing/what-are-fleet-policies), [queries](https://fleetdm.com/guides/queries), and [configuration profiles](https://fleetdm.com/guides/custom-os-settings) to specific hosts. In addition, you can use labels to create a custom filtered view of your hosts.

Reviewer: ```suggestion
In Fleet, you can add labels (tags) to specific hosts to scope [software](https://fleetdm.com/guides/deploy-software-packages), [policies](https://fleetdm.com/securing/what-are-fleet-policies), [queries](https://fleetdm.com/guides/queries), and [configuration profiles](https://fleetdm.com/guides/custom-os-settings). You can also use labels to create a custom, filtered view of your hosts.
```

Reviewer: Rephased for ease of reading (at least for me). I also removed "(tags)." I feel like tags and labels are more or less the same thing.

> Labels in Fleet can be on of the following types:

Reviewer: ```suggestion
Labels can be one of the following types:
```

- **Dynamic**: Query based label. All hosts that return a result to the query get this label applied.
- **Manual**: A list of selected hosts.
- **Host vitals**: All hosts that have a specific host vital get this label applied. Currently only supported for IdP host vitals (groups and department) on macOS, iOS, iPadOS, and Android hosts.

> To add or edit a label in Fleet, select the avatar  select the avatar on the right side of the top navigation and select **Labels**.

Reviewer: ```suggestion
To add or edit a label in Fleet, select the avatar on the right side of the top navigation and select **Labels**.
```

You can also manage labels via [Fleet's API](https://fleetdm.com/docs/rest-api/rest-api#labels) or [best practice GitOps](https://fleetdm.com/docs/configuration/yaml-files#labels).

> > For dynamic labels, if you want to change the query or platforms, you must delete the existing label and create a new one.

Reviewer: ```suggestion
> For dynamic labels, if you want to change the query or platform, you must delete the existing label and create a new one.
```
